### DISCUSSION TITLE
so-filebeat missing on forwarder

### DISCUSSION THREAD

**Discussion by ghostrider9899** [Security-Onion-Solutions/securityonion#8044] [Unsupported Versions]
How can i redownload the so-filebeat container for my forwarder node? I tried reinstalling the software and its now reporting the so-filebeat missing when i do a so-status

**Comment by dougburks** [comment]
Try `sudo so-docker-refresh`.

**Reply by Singularity-Zed** [reply]
Any alternative to sudo so-docker-refresh for airgapped systems having a similar issue?

**Reply by dougburks** [reply]
@Singularity-Zed From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
